api/src/stampy_chat/chat.py

```diff
@@ -26,6 +28,9 @@
 
 ENCODER = tiktoken.get_encoding("cl100k_base")
 
+# initialize a budget manager to control costs for gpt-4/other llms
+budget_manager = litellm.BudgetManager(project_name="stampy_chat")
```

Comment:
how does the budget per user get configured? Could you add a new item to `env.py` so that it can be configured? Also, what would the units be? (I had a very quick glance at the litellm docs, but otherwise don't know anything about it)

Reply:
(I'll give it a proper look tomorrow)